babel.config.js

```diff
@@ -0,0 +1,28 @@
+const presets = [
+    [
+        "@babel/preset-env",
+        {
+            targets: {
+                ie: "10",
+                node: "current"
```

Comment:
Does this always point to the latest version of node as it is released? Should we specify 10?